builder/config_reader_test.go

```diff
@@ -108,30 +108,31 @@ func testConfig(t *testing.T, when spec.G, it spec.S) {
 				})
 			})
 
-			when("'uri' is misspelled as url", func() {
+			when("unknown buildpack key is present", func() {
```

Comment:
Made this more consistent with https://github.com/buildpacks/pack/blob/30cb3dca7c746d1b21c00dc550f5b8219d3a7b9f/buildpackage/config_reader_test.go#L133. In general the test descriptions should describe the general category of what is being tested.